sensor_msgs/NavSatFix desired_point
float32 K_1 # 0.0-1.0
float32 K_2 # 0.0-1.0
float32 K_3 # 0.0-1.0
uint8   gentleness # scale 0-100
float32 move_accuracy # meters
float32 rotation_accuracy # degrees